resources/java_memory_assistant/bin/clean-up.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh -e
```

Comment:
Because the Java memory assistant is something to be used also in places other than the community Java buildpack. Besides, as pretty much any agent goes, the less it contains in terms of code, the far better it is.

Besides, agents should rather not use 3rd party line to avoid as much as possible nasty classloading conflicts. So, instead of baking stuff in the Java memory agent, we made a set of hooks that allows to specify commands to be execute. As an extra benefit, the footprint of the command does not impact the JVM, but the OS directly (Runtime API).